openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -78,6 +80,22 @@ def _gather_graded_items(root, due):  # lint-amnesty, pylint: disable=missing-fu
     return []
 
 
+def _get_custom_pacing_children(subsection, num_weeks):
+    """
+    Return relative date items for the subsection and its children
+    """
+    items = [subsection]
+    section_date_items = []
+    while items:
+        next_item = items.pop()
+        if next_item.category != 'openassessment':
+            section_date_items.append((next_item.location, {'due': timedelta(weeks=num_weeks)}))
+            items.extend(next_item.get_children())
+        else:
+            return []
```

Comment:
Hmm. What happens if there is an openassessment and other content within the same subsection? Will this early exit incorrectly? It looks like above, as long as there was some item that was not an ORA (Open Response Assessment)